### ISSUE TITLE
PUT request not detected

### ISSUE THREAD

**Issue by r-ryanwatson** [snowflakedb/snowflake-connector-nodejs#242]
Please answer these questions before submitting your issue. Thanks!

1. What version of NodeJS are you using (`node --version` and `npm --version`)?
17.1.0

2. What operating system and processor architecture are you using?
macOS 10.15.7 

3. What are the component versions in the environment (`npm list`)?
snowflake-sdk@1.6.5

4. What did you do?

I used:
```
connection.execute({ 
    sqlText: `    
        PUT 'file:///Users/file.csv' @%MY_TABLE
        OVERWRITE = TRUE`,
    complete: (e) => console.log(e)
});
```
as a PUT request.

5. What did you expect to see?
Util.isPutCommand(options.sqlText) determine that the request I made is a PUT request.

6. What did you see instead?
"Unsupported feature 'unsupported_requested_format:snowflake'."

A template literal with a newline and possibly some space before the word PUT will not be detected as a PUT command with the following code:
```
exports.isPutCommand = function (sqlText)
{
  return (sqlText.substring(0, 3).toUpperCase() === "PUT");
};
```

**Comment by sfc-gh-dszmolka** [comment]
thank you for reporting this issue. it seems to be still an issue in the current (v1.6.17) version of the connector. we'll take a look.

**Comment by sfc-gh-dszmolka** [comment]
#400

**Comment by sfc-gh-dszmolka** [comment]
fix released with version [1.6.22](https://github.com/snowflakedb/snowflake-connector-nodejs/releases/tag/v1.6.22)
